include/apple2enh.h

```diff
@@ -102,6 +102,7 @@ extern void a2e_stdmou_mou[];     /* Referred to by mouse_static_stddrv[] */
 extern void a2e_ssc_ser[];        /* Referred to by ser_static_stddrv[]   */
 extern void a2e_hi_tgi[];         /* Referred to by tgi_static_stddrv[]   */
 extern void a2e_lo_tgi[];
+extern void a2gs_ssc_ser[];        /* IIgs serial driver   */
```

Comment:
This needs to be named `a2e_scc_ser`:
* `a2e` refers to the cc65 target name, not to the actual machine
* `ssc` vs. `scc` ;-)

Reply:
I didn't even plan on building it for __APPLE2__ non-enhanced, I think there's no reason anyone would want to build the IIgs driver into a non-enhanced build? But I could do it if you think it has an interest.

Reply:
See my by now added general comment.